packages/editor-ui/src/components/SettingsLogStreaming/EventDestinationCard.ee.vue

```diff
@@ -130,17 +128,16 @@ export default defineComponent({
 				);
 			}
 		},
-		async onClick(event?: PointerEvent) {
+		async onClick(event: Event) {
 			if (
-				event &&
-				event.target &&
-				'className' in event.target &&
-				event.target['className'] === 'el-switch__core'
+				this.$refs.cardActions === event.target ||
+				this.$refs.cardActions?.contains(event.target) ||
+				event.target?.contains(this.$refs.cardActions)
```

Comment:
This condition could run into a situation where the click event is cancelled even when the event is triggered by clicking the card itself (the wrapper does contain the card actions).

To avoid this, you could add `height: 100%` to the `cardActions` class and then align child items to the center.

Reply:
And actually with that in place we can also remove `event.target?.contains(this.$refs.cardActions)` because the outer most target can only be `cardActions` which is already checked by `this.$refs.cardActions === event.target`